Godeps/Godeps.json

```diff
@@ -34,8 +33,8 @@
 		},
 		{
 			"ImportPath": "github.com/opencontainers/runtime-spec/specs-go",
-			"Comment": "v1.0.0-rc1-31-gbb6925e",
-			"Rev": "bb6925ea99f0e366a3f7d1c975f6577475ca25f0"
+			"Comment": "v1.0.0-rc5-30-g3adac26",
```

Comment:
I now use `godep update` to update the `runtime-spec`, after the update directly to the latest version (`v1.0.0-rc5-30`), what method can you only be updated to `1.0.0-rc5`? Thanks.